generate-test-vector.py

```diff
@@ -854,120 +534,23 @@ def generate_change_tests():
             'scriptSig': get_p2pkh_scriptsig(input_pub_keys[i], input_priv_keys[i][0]),
             'txinwitness': '',
             'prevout': {'scriptPubKey': {'hex': get_p2pkh_scriptPubKey(input_pub_keys[i])}},
+            'private_key': input_priv_keys[i][0].get_bytes().hex(),
         }]
 
-    sender['given']['vin'] = add_private_keys(deepcopy(inputs), input_priv_keys)
-    sender['given']['recipients'] = addresses
-    A_sum = sum(input_pub_keys)
-    deterministic_nonce = reference.get_input_hash([COutPoint(deser_txid(o[0]), o[1]) for o in outpoints], A_sum)
-    outputs = reference.create_outputs(input_priv_keys, deterministic_nonce, addresses, hrp=HRP)
-    sender['expected']['outputs'] = outputs
-
-
-    test_case['sending'].extend([sender])
-    labels = [change_labels, []]
-    for i, rec in enumerate([rec1, rec2]):
-        rec['given']['vin'] = inputs
-        rec['given']['outputs'] = outputs
-
-        scan, spend, Scan, Spend = derive_silent_payment_key_pair(bytes.fromhex(seeds[i]))
-        add_to_wallet = reference.scanning(
-            scan,
-            Spend,
-            A_sum,
-            deterministic_nonce,
-            [ECPubKey().set(bytes.fromhex(pub)) for pub in outputs],
-            labels={(reference.generate_label(scan, l)*G).get_bytes(False).hex():reference.generate_label(scan, l).hex() for l in labels[i]},
-        )
-        for o in add_to_wallet:
-
-            pubkey = ECPubKey().set(bytes.fromhex(o['pub_key']))
-            full_private_key = spend.add(
-                bytes.fromhex(o['priv_key_tweak'])
-            )
-            if full_private_key.get_pubkey().get_y()%2 != 0:
-                full_private_key.negate()
-
-            sig = full_private_key.sign_schnorr(msg, aux)
-            assert pubkey.verify_schnorr(sig, msg)
-            o['signature'] = sig.hex()
-
-        rec['expected']['outputs'] = add_to_wallet
-        test_case['receiving'].extend([rec])
-    test_case["comment"] = "Single recipient: use silent payments for sender change"
-    test_cases.append(test_case)
-    return test_cases
-
-
-# In[17]:
-
-
-def generate_unknown_segwit_ver_test():
```

Comment:
in 404fe33e9318f200959e76875505edec7be10e90

nit: should we still have a test for unknown segwit version?